lib/raffle/ui/pages/main_page/main_page.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: unnecessary_to_list_in_spreads
+
```

Comment:
Why do we ignore the linting here ?

Reply:
Woups. Too fast on the quick fix button.